### PR TITLE
[MU4] Added unity build

### PR DESCRIPTION
**Pull request by igorkorsukov**
on my machine build speed up from 04:05 to 02:18 (without ccache)

Gave a lot of problems `using namespace Ms`, I strongly discourage using it anyway.

### CODE REVIEW

File: src/context/internal/globalcontext.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ using namespace mu::notation;
 using namespace mu::shortcuts;
 using namespace mu::async;
 
-static const mu::Uri NOTAION_PAGE("musescore://notation");
+static const mu::Uri NOTAION_PAGE_URI("musescore://notation");
```
Comment (Jojo-Schmitz) [inline src/context/internal/globalcontext.cpp:26]: Really NOTAION, not NOTA**T**ION?

File: src/context/internal/globalcontext.cpp [hunk 2]
```diff
@@ -101,7 +101,7 @@ ShortcutContext GlobalContext::currentShortcutContext() const
 {
     if (playbackController()->isPlaying()) {
         return ShortcutContext::Playing;
-    } else if (interactive()->currentUri().val == NOTAION_PAGE) {
+    } else if (interactive()->currentUri().val == NOTAION_PAGE_URI) {
```
Comment (Jojo-Schmitz) [inline src/context/internal/globalcontext.cpp:104]: here too

File: src/context/internal/globalcontext.cpp (hunk 1 above)
Comment (igorkorsukov) [reply src/context/internal/globalcontext.cpp:26]: Thanks, I will fix in next PR

Comment (Jojo-Schmitz) [reply src/context/internal/globalcontext.cpp:26]: really a minor nitpick ;-)

**Pull request merged**